src/libraries/System.Drawing.Common/src/System/Drawing/Bitmap.Unix.cs

```diff
@@ -73,7 +73,7 @@ public Bitmap(Type type, string resource)
             if (type == null)
                 throw new NullReferenceException();
 
-            Stream s = type.GetTypeInfo().Assembly.GetManifestResourceStream(type, resource);
+            Stream? s = type.GetTypeInfo().Assembly.GetManifestResourceStream(type, resource);
```

Comment:
Hmm interesting, in Unix, we throw `FileNotFoundException` and in windows we throw `ArgumentException` if the resource was not found. Should we fix the Unix one to be `ArgumentException`? cc: @stephentoub @JeremyKuhne

Reply:
I don't see that we [document](https://docs.microsoft.com/en-us/dotnet/api/system.drawing.bitmap.-ctor?view=netcore-3.1#System_Drawing_Bitmap__ctor_System_Type_System_String_) what goes on here. I'd say we should go with `ArgumentException`.

The parameter checking bothers me. I'm inclined to see it changed to `ArgumentNullException` for both arguments and documenting the `NullRef` for Framework.

Reply:
Not sure if we need/want to address that with this PR